tests/storage/golden_image/test_golden_image.py

```
@@ -97,53 +84,72 @@ def test_regular_user_cant_create_dv_in_ns(
 @pytest.mark.sno
+@pytest.mark.parametrize(
+    "golden_image_data_volume_scope_module",
+    [
+        pytest.param(DV_PARAM, marks=pytest.mark.polarion("CNV-4756")),
+    ],
+    indirect=True,
+)
 @pytest.mark.s390x
-@pytest.mark.polarion("CNV-4756")
 def test_regular_user_cant_delete_dv_from_cloned_dv(
+    golden_images_namespace,
```

Ruff (0.16.1)

[warning] Unused function argument: `golden_images_namespace`

(ARG001)